packages/wallets/src/features/cfd/types.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+    useAvailableMT5Accounts,
+    useCreateOtherCFDAccount,
+    useCtraderAccountsList,
+    useDxtradeAccountsList,
+    useMT5AccountsList,
+    useSortedMT5Accounts,
+} from '@deriv/api';
+
+// eslint-disable-next-line  @typescript-eslint/no-namespace
+export namespace THooks {
+    export type AvailableMT5Accounts = NonNullable<ReturnType<typeof useAvailableMT5Accounts>['data']>[number];
+    export type CtraderAccountsList = NonNullable<ReturnType<typeof useCtraderAccountsList>['data']>[number];
+    export type DxtradeAccountsList = NonNullable<ReturnType<typeof useDxtradeAccountsList>['data']>[number];
+    export type MT5AccountsList = NonNullable<ReturnType<typeof useMT5AccountsList>['data']>[number];
+    export type SortedMT5Accounts = NonNullable<ReturnType<typeof useSortedMT5Accounts>['data']>[number];
```

Comment:
any particular reason to have separate type for "sorted" accounts? it feels more like an array property itself and less than object type itself. We might end up with types like "AllMT5Accounts", "CryptoMT5Accounts" and every single variation.